changelogs/fragments/549-fix-server-side-apply.yaml

```diff
@@ -1,3 +1,4 @@
 ---
 bugfixes:
   - k8s - Fix issue with server side apply with kubernetes release '25.3.0' (https://github.com/ansible-collections/kubernetes.core/issues/548).
+  - k8s - Fix issue with server side apply with kubernetes release '25.3.0' (https://github.com/ansible-collections/kubernetes.core/issues/547).
```

Comment:
Could you change the description to address that this fixes check mode? This problem does not have anything to do with the kubernetes version.

Reply:
Copy/paste issue. Fixed
Thanks